ui/workshop.css

```
@@ -0,0 +1,394 @@
+#page-workshop #upGame .fmenu { z-index:80; max-height:220px; overflow-y:auto; }
+#page-workshop #upGame.gopen .fmenu { display:block; }
+#page-workshop #upGame.gopen { border-color:rgba(var(--accent-rgb),.5); }
+#page-workshop .up-seg { display:flex; gap:3px; background:var(--card-inner); border:1px solid var(--glass-border); border-radius:10px; padding:3px; }
+#page-workshop .up-seg-btn { flex:1; padding:7px 10px; border:none; background:none; color:var(--text-muted);
+               font-family:inherit; font-size:12px; font-weight:500; border-radius:7px; cursor:pointer; }
+#page-workshop .up-seg-btn.active { background:rgba(var(--accent-rgb),.18); color:var(--purple-l2); }
+#page-workshop .up-chips { display:flex; flex-wrap:wrap; gap:8px; }
+#page-workshop .up-chip { display:inline-flex; align-items:center; gap:7px; font-family:inherit; font-size:12px; font-weight:500;
+            padding:7px 13px; border-radius:99px; cursor:pointer;
+            border:1px solid var(--glass-border); background:rgba(255,255,255,.04); color:var(--text-muted); }
+#page-workshop .up-chip .d { width:8px; height:8px; border-radius:50%; background:currentColor; opacity:.5; }
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)